fvm/fvm_blockcontext_test.go

```diff
@@ -25,6 +25,7 @@ import (
 	errors "github.com/onflow/flow-go/fvm/errors"
 	"github.com/onflow/flow-go/fvm/storage/snapshot"
 	"github.com/onflow/flow-go/model/flow"
+	pmock "github.com/onflow/flow-go/state/protocol/mock"
```

Comment:
It is common practice in our code base to name the mock by exactly the same package as the original. 
 ```suggestion
	protocol "github.com/onflow/flow-go/state/protocol/mock"
```

Reply:
That's not always possible because the non-mock package is also imported and we can't import two different packages with the same name. 
I renamed to `protocolMock` for more clarity.